src/connectors/index.ts

```diff
@@ -50,8 +50,8 @@ export const walletconnect = {
     qrcode: true,
     pollingInterval: POLLING_INTERVAL,
   }),
-  4: new WalletConnectConnector({
-    rpc: { 4: NETWORK_CONFIGS[4].rpc },
+  5: new WalletConnectConnector({
```

Comment:
This change will remove support for rinkeby...I think add a new option for Goerli and leave chain id 4 as it is.

Reply:
Right, the previous commit in this branch mentioned that Rinkeby support was to be removed in replace of Goerli. There was no reference to Rinkeby in the code apart from this file.